tshirts.py

```diff
@@ -8,7 +8,9 @@ def size(cms):
         return 'L'
 
 
-assert(size(37) == 'S')
-assert(size(40) == 'M')
-assert(size(43) == 'L')
+assert(get_size(37) == 'S')

```

Comment:
All boundary conditions must be addressed - 38, 42 and values in between. These tests can be enhanced as an improvement

Reply:
@prithamvshetty Done feedback changes

Reply:
👍